receiver/opencensusreceiver/testdata/config.yaml

```diff
@@ -1,19 +1,54 @@
 receivers:
+  # The following entry initializes the default OpenCensus receiver.
+  # The default values are specified here
+  #  https://github.com/open-telemetry/opentelemetry-service/blob/71589202609d7e787244076b631b45e219101867/receiver/opencensusreceiver/factory.go#L47-L56
+  # The full name of this receiver is `opencensus` and can be referenced in pipelines by 'opencensus'.
   opencensus:
+  # The following entry demonstrates configuring the common receiver settings:
+  # - endpoint
+  # - disable-backpressure
+  # For more information on the struct, refer to
+  # https://github:com/open-telemetry/opentelemetry-service/blob/71589202609d7e787244076b631b45e219101867/config/configmodels/configmodels.go#L142-L150
+  # This configuration has a name 'customname' with a full name of 'opencensus/customname'.
+  # To reference this configuration in a pipeline, use the full name `opencensus/customname`.
   opencensus/customname:
+    # The endpoint for this receiver will be: "0.0.0.0:9090".
```

Comment:
I think it will be useful to mention that this is the endpoint the receiver will _listen_ on.